connector/src/main/java/org/geysermc/connector/utils/SkinProvider.java

```diff
@@ -111,6 +112,21 @@
 
         EARS_GEOMETRY_SLIM = earsDataBuilder.toString();
 
+        /* Load in the custom skull geometry */
+        InputStream skullStream = FileUtils.getResource("bedrock/skin/geometry.humanoid.customskull.json");
+
+        StringBuilder skullDataBuilder = new StringBuilder();
+        try (Reader reader = new BufferedReader(new InputStreamReader(skullStream, Charset.forName(StandardCharsets.UTF_8.name())))) {
+            int c = 0;
+            while ((c = reader.read()) != -1) {
+                skullDataBuilder.append((char) c);
+            }
+        } catch (IOException e) {
+            throw new AssertionError("Unable to load skull geometry", e);
+        }
```

Comment:
Would've used Files#readAllBytes here.

Reply:
All other file reads in the class use this method, so at least it's consistent.